admin_guide/advanced_scheduling/pod_affinity.adoc

```diff
@@ -0,0 +1,351 @@
+
+In this example, the pod affinity rule indicates that the pod can schedule onto a node only if that node has at least one already-running pod with a label that has the key `security` and value `S1`. The pod anti-affinity rule says that the pod prefers to not schedule onto a node if that node is already running a pod with label having key `security `and value `S2`.
+
+.Example pod config file with pod affinity
+====
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: with-pod-affinity
+spec:
+  affinity:
+    podAffinity: <1>
+      requiredDuringSchedulingIgnoredDuringExecution: <2>
+      - labelSelector:
+          matchExpressions:
+          - key: security <3>
+            operator: In <4>
+            values:
+            - S1 <3>
+        topologyKey: failure-domain.beta.kubernetes.io/zone
+  containers:
+  - name: with-pod-affinity
+    image: gcr.io/google_containers/pause:2.0
+
+----
+====
+<1> Stanza to configure pod affinity.
+<2> Defines a required rule.
+<3> The key and value (label) that must be matched to apply the rule.
+<4> The xref:../../rest_api/openshift_v1.html#unversioned-labelselectorrequirement[operator] represents the relationship between the key and a set of values.  Can be `In`, `NotIn`, `Exists`, or `DoesNotExist`.
+
+.Example pod config file with pod anti-affinity
+====
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: with-pod-affinity
+spec:
+  affinity:
+    podAntiAffinity: <1>
+      preferredDuringSchedulingIgnoredDuringExecution: <2>
+      - weight: 100 <3>
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+            - key: security <4>
+              operator: In <5>
+              values:
+              - S2 <4>
+          topologyKey: kubernetes.io/hostname
+  containers:
+  - name: with-pod-affinity
+    image: gcr.io/google_containers/pause:2.0
+
+----
+====
+<1> Stanza to configure pod anti-affinity.
+<2> Defines a preferred rule.
+<3> Specifies a weight for a preferred rule. The node that with highest weight is preferred.
+<4> The key and value (label) that must be matched to apply the rule.
+<5> The xref:../../rest_api/openshift_v1.html#unversioned-labelselectorrequirement[operator] represents the relationship between the key and a set of values.  Can be `In`, `NotIn`, `Exists`, or `DoesNotExist`.
+
+
+[NOTE]
+====
+If labels on a node change at runtime such that the affinity rules on a pod are no longer met, the pod will still continue to run on the node.
+====
+
+[[admin-guide-sched-affinity-config-pod-req]]
+=== Configuring an Affinity Rule
+
+The following steps demonstrate a simple two-pod configuration that creates pod with a label and a pod that uses affinity to allow scheduling with that pod.
+
+. Create a pod with a specific label in the pod specification:
++
+----
+$ cat team4.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: team4
+  labels:
+     team: "4"
+spec:
+  containers:
+  - name: ocp
+    image: docker.io/ocpqe/hello-pod
+----
```

Comment:
for matching step 3, you might want use this to replace team4.yaml
cat pod-s1.yaml
```
apiVersion: v1
kind: Pod
metadata:
  name: security-s1
  labels:
    security: S1
spec:
  containers:
  - name: security-s1
    image: docker.io/ocpqe/hello-pod
```